CONTRIBUTING.md

```diff
@@ -0,0 +1,69 @@
+# Contributing
+All kind of contribution is welcome: reporting new issues, pull requests or tweeting about this project.
+
+
+## Installation
+In this section we will focus to setup the project to run the tests in our local environment. 
+
+Orator support many python versions and also multiple databases so the setup can be a little difficult. To make it 
+as easy as possible we will use docker.
+
+But before beginning there are some prerequisites to setup the environment.
+
+### Prerequisites
+  * [docker](https://docs.docker.com/install/)
+  * [pyenv](https://github.com/pyenv/pyenv)
+  * [poetry](https://github.com/sdispater/poetry)
+
+### Instructions
+
+#### Python 3.7
+First we will install python 3.7 using pyenv. But before installing this version of python you will need some
+packages installed to compile python correctly. You can install them using apt.
+
+  * libsqlite3-dev
```

Comment:
@psincraian  Also if you're going by the docker route shouldn't we just make one container  and install eveything in that container? in that case even these dependencies can go inside docker?

Reply:
I will try to dockerize it 😊